pkgs/applications/graphics/gltfpack/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, nixos
+, cmake
+, meshoptimizer
+, gltfpack
+}:
+let basis_universal = fetchFromGitHub {
+      owner = "zeux";
+      repo = "basis_universal";
+      rev = "8903f6d";
+      hash = "sha256-o3dCxAAkpMoNkvkM7qD75cPn/obDc/fJ8u7KLPm1G6g=";
+    };
+in
+stdenv.mkDerivation rec {
+  name = "gltfpack";
+  version = "0.18";
+  src = fetchFromGitHub {
+    owner = "zeux";
+    repo  = "meshoptimizer";
+    hash  = "sha256-OWeptdnKFvTyfkz0sFCpiTI7323GfVE8vb8bNUBnslA=";
+    rev   = "49d9222";
+  };
+
+  nativeBuildInputs = [ meshoptimizer cmake ];
```

Comment:
if `meshoptimizer` is needed at runtime, then we have to put it in `buildInputs`. `cmake` can stay here, since it is only needed at build time.